src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -284,4 +287,12 @@ private function getContext(string $template, int $line, int $context = 3)
 
         return $result;
     }
+
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        if ($input->mustSuggestOptionValuesFor('format')) {
+            $helper = new DescriptorHelper();
+            $suggestions->suggestValues($helper->getFormats());
```

Comment:
This command does not use this helper. Supported formats are specified in the [`display` method](https://github.com/symfony/symfony/pull/43680/files#diff-eef83d0198d3a7daee9a487dbaad38bd11b7be712fd0c65ab3b344a97cdaad83R186-R194) of this class.
```suggestion
            $suggestions->suggestValues(['txt', 'json', 'github']);
```